core/deployment/src/main/java/io/quarkus/deployment/pkg/NativeConfig.java

```diff
@@ -132,7 +133,7 @@
     /**
      * The docker image to use to do the image build
      */
-    @ConfigItem(defaultValue = "quay.io/quarkus/ubi-quarkus-native-image:19.2.1")
+    @ConfigItem(defaultValue = "quay.io/quarkus/ubi-quarkus-native-image:19.3.1-java8")
```

Comment:
Should we start using Java 11 here?

Reply:
I don't think so since it still experimental isn't it?

Reply:
That's fine by me, maybe flip the switch when we require JDK 11 as the minimum version in the future

Reply:
Makes sense